docs/cluster_spec.md

```diff
@@ -889,6 +889,16 @@ docker:
     - "dm.use_deferred_removal=true"
 ```
 
+### networking
+
+To run `docker run` commands with network and internet access, you need to enable the necessary [iptables](https://docs.docker.com/network/iptables/) rules:
```

Comment:
```suggestion
In order for containers started with `docker run` instead of Kubernetes to have network and internet access you need to enable the necessary [iptables](https://docs.docker.com/network/iptables/) rules:
```